src/client/core/utils/content-editable.js

```diff
@@ -210,8 +214,15 @@ export function isInvisibleTextNode (node) {
     return firstVisibleIndex === nodeValue.length && lastVisibleIndex === 0;
 }
 
+function checkTextNodeVisibility (node, onlyVisible) {
```

Comment:
For me it seems little awkward that method named `checkTextNodeVisibility` can do not check visibility if we pass `onlyVisible === false` .
@AndreyBelym